### PR TITLE
web: codemod `web/global/GlobalDebug` `web/Layout` `web/user/settings/UserSettingsSidebar` `user/settings/emails/UserSettingsEmailsPage` components to CSS modules

### PR DESCRIPTION
**Pull request by gitstart-sourcegraph**
## Description
This PR results from the global-css-to-css-modules [codemod](https://github.com/sourcegraph/codemod/tree/main/src/transforms/globalCssToCssModule) application with some manual fixes, caused by the usage of global CSS classes tied to a React component in other parts of the codebase.

## Refs

- [SG Issue](https://github.com/sourcegraph/sourcegraph/issues/26700)
- [Gitstart Ticket](https://app.gitstart.com/clients/sourcegraph/tickets/SG-26700)

## Success criteria

The stylesheets in the following directories are converted to CSS modules.

- client/web/src/global/GlobalDebug.tsx
- client/web/src/Layout.tsx
- client/web/src/user/settings/UserSettingsSidebar.tsx
- client/web/src/user/settings/emails/UserSettingsEmailsPage.tsx

### CODE REVIEW

**Comment by valerybugakov** [comment]
@gitstart-sourcegraph, is this PR still in progress?

**Review by valerybugakov — COMMENTED** [review]
@gitstart-sourcegraph, is this PR still in progress?

**Comment by gitstart-sourcegraph** [comment]
@valerybugakov We see that the class `global-debug` is used in `client/browser` package and we couldn't find the way to see those UI on app/extensions, so we want to confirm that
- Should we replace `global-debug` usages in `client/browser` package with `global-debug` attribute?
- Could you show us how to see UI related to `global-debug` on UI

**Review by valerybugakov — COMMENTED** [review]
@gitstart, let's replace the `global-debug` class with the data attribute in the `browser` package. To see the UI of the browser extension follow the steps defined [here](https://github.com/sourcegraph/sourcegraph/blob/main/client/browser/README.md#development),

File: client/web/src/user/settings/UserSettingsArea.scss [hunk 1]
```diff
@@ -1,5 +1,9 @@
 @import './profile/UserProfileFormFields';
 @import './profile/UserSettingsProfilePage.scss';
 @import './repositories/UserSettingsRepositoriesPage';
-@import './emails/UserSettingsEmailsPage.scss';
-@import './UserSettingsSidebar';
+
+// Move these rules from UserSettingsSidebar.scss in css module migrations ofUserSettingsSidebar,
```
Comment (valerybugakov) [inline client/web/src/user/settings/UserSettingsArea.scss:5]:
```suggestion
// Move these rules from UserSettingsSidebar.scss in css module migrations of UserSettingsSidebar,
```

**Review by valerybugakov — APPROVED** [review]

**Pull request merged**